kuka_kr5_support/test/roslaunch_test.xml

```diff
@@ -1,9 +1,9 @@
 <launch>
```

Comment:
Not sure if this file will work as it is. We should do another way of launch tests that is ROS 2 compatible. Please check in internet for an example and make a proposal.

Reply:
but if looked at roslaunch_test.xml there is no test tag and i dont see any tests happening, 

but if we want to include some tests the i found this launch_testing for ros2